Osushi/View/PostList/DetailPostView.swift

```diff
@@ -39,6 +39,9 @@ struct DetailPostView: View {
                         tappedFavoriteButton()
                     }
                     .popoverTip(FavoriteButtonTip())
+                    .sensoryFeedback(.impact(weight: .light), trigger: isFavorited) { _, new in
```

Comment:
とても良いです！ありがとうございます。
OFF にした場合も発動させた方がもっと良くなると思うので、以下のように修正お願いします！

```suggestion
.sensoryFeedback(.impact, trigger: isFavorited)
```